commcare_export/minilinq.py

```diff
@@ -477,9 +477,9 @@ def __init__(
         table: str,
         headings: ListType[MiniLinq],
         source: MiniLinq,
-        missing_value=None,
-        data_types=None,
-    ):
+        missing_value: Optional[str] = None,  # Denotes "no value". e.g. "---"
```

Comment:
style question... should this be here or in the docstring? I don't think I've seen per argument comments like this before.

Reply:
Great question. I was unsure.

I'll create a docstring for `__init__()`. Some of those other params could also use a bit of explaining.